device/mellanox/x86_64-nvidia_sn2201-r0/platform_components.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
if this is for the fw util and we dont support it yet, should we add it?

Reply:
Done, confirmed with Nazari, file removed.